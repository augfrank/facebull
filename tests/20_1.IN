M0   C0   C2  2549
M1   C0   C4  2960
M2   C0   C6  2079
M3   C0   C14  2041
M4   C0   C16  2034
M5   C0   C18  2389
M6   C1   C0  2864
M7   C1   C10  2556
M8   C1   C15  2167
M9   C2   C6  2313
M10   C2   C14  2047
M11   C3   C0  2264
M12   C3   C9  2173
M13   C3   C12  2868
M14   C4   C3  2560
M15   C4   C13  2631
M16   C4   C14  2978
M17   C5   C2  2619
M18   C5   C14  2118
M19   C5   C17  2353
M20   C5   C19  2934
M21   C6   C5  2381
M22   C6   C8  2207
M23   C6   C12  2006
M24   C7   C12  2849
M25   C7   C19  2050
M26   C8   C7  2149
M27   C8   C10  2292
M28   C9   C3  2025
M29   C9   C13  2528
M30   C9   C16  2721
M31   C9   C19  2920
M32   C10   C0  2554
M33   C10   C17  2423
M34   C11   C15  2217
M35   C12   C3  2267
M36   C12   C7  2551
M37   C12   C8  2217
M38   C12   C11  2788
M39   C13   C6  2618
M40   C13   C12  2739
M41   C14   C7  2706
M42   C14   C10  2941
M43   C14   C16  2832
M44   C15   C1  2616
M45   C15   C5  2161
M46   C15   C6  2977
M47   C15   C9  2941
M48   C16   C4  2425
M49   C16   C8  2346
M50   C16   C13  2027
M51   C16   C15  2031
M52   C17   C18  2549
M53   C18   C6  2172
M54   C18   C9  2441
M55   C18   C13  2681
M56   C19   C0  2633
M57   C19   C1  2658
M58   C19   C7  2063
M59   C19   C12  2720
