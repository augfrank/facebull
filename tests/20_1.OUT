48088
3 7 9 11 15 17 22 25 26 28 33 34 38 40 43 45 48 52 54 57
